src/test/ui/parser/keyword-const-as-identifier.rs

```diff
@@ -1,5 +0,0 @@
-// This file was auto-generated using 'src/etc/generate-keyword-tests.py const'
-
-fn main() {
-    let const = "foo"; //~ error: expected identifier, found keyword `const`
```

Comment:
we should keep this test imo, just to track how we're reporting the error on it.

If you want the original test back, you can use `struct const;`, which should give the same error

Reply:
Yeah, test output just needs to be updated.